frontend/public/style/coreos.css

```diff
@@ -145,6 +145,10 @@ input[type="radio"] {
   position: relative;
   width: 100%; }
 
+.co-m-row:hover {
+  background-color: #f8f9fa;
```

Comment:
We should try one of the grays from the Patternfly palette, maybe `#fafafa` or `#f5f5f5`.

https://www.patternfly.org/styles/color-palette/

Reply:
> We should try one of the grays from the Patternfly palette, maybe `#fafafa` or `#f5f5f5`.
> 
> https://www.patternfly.org/styles/color-palette/

@serenamarie125

Reply:
@alecmerdler given issues with our previous blues, i agree that #f5f5f5 is good for this PR.  Thank you!